business/src/main/java/org/mskcc/cbio/portal/model/DBMutationData.java

```diff
@@ -36,4 +36,8 @@
     public String variant_read_count_normal;
     public String cosmic_count;
     public String keyword;
+    public String binary_filter;
+    public String binary_filter_annotation;
+    public String multi_filter;
+    public String multi_filter_annotation;
```

Comment:
I'm not a fan of these variables names, (see other comment). Can also change in a later iteration of this PR.